src/phases/query/QueryStatsStages.tmpl

```diff
@@ -0,0 +1,161 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of helper stages for QueryStats workloads. Each stage represents a single
+  workload variant, comprising multiple phases. Multiple stages can be grouped together if they have
+  commonalities in their variations.
+
+GlobalDefaults:
+  # Workload "constants."
```

Comment:
Some of these seem to be repeated here and in the actors? DB and Collection. If this is required then it's OK, but a comment about how they need to be in sync between the two places is probably worthwhile.